Common/Sources/Extensions/HyperwalletTransferMethodType.swift

```diff
@@ -31,9 +31,9 @@ public extension HyperwalletTransferMethodType {
         if let fees = self.fees?.nodes {
             let feeLabel = "add_transfer_method_fee_label".localized()
 
-            attributedText.append(value: feeLabel, font: Theme.Label.captionOne, color: color)
+            attributedText.append(value: feeLabel, font: UIFont.preferredFont(forTextStyle: .caption1), color: color)
             attributedText.append(value: HyperwalletFee.format(fees: fees),
-                                  font: Theme.Label.captionOneMedium,
+                                  font: UIFont.preferredFont(forTextStyle: .caption1),
```

Comment:
This should still be Theme.Label.captionOneMedium. Update Theme class to assign ``` UIFont.preferredFont(forTextStyle: .caption1) ``` to the Theme.Label.captionOneMedium

Reply:
Modified